src/backend/functions/change-requests.ts

```diff
@@ -0,0 +1,37 @@
+import { Context } from 'aws-lambda';
+import { ChangeRequest } from 'utils/src/types/change-request-types';
+
+interface ChangeRequestResponse {
+  statusCode: number;
+  headers?: any;
+  body?: string;
+}
+
+export async function handler(event: any, context: Context): Promise<ChangeRequestResponse> {
+  try {
+    return {
+      statusCode: 200,
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(exampleCRs)
+    };
+  } catch (error) {
+    console.error(error);
+    return { statusCode: 500 };
+  }
+}
+
+const changeRequest1: ChangeRequest = {
```

Comment:
These fields don't look correct, are you sure the type-checking is happy with it? If yes, we'll definitely need to wait until my massive PR gets merged before merging this one too.

Reply:
These should be swapped out for the ones from the `dummy-data.ts` file from the `utils` package